src/main/java/InputParser.java

```diff
@@ -0,0 +1,83 @@
+public class InputParser {
+    protected String[] args;
+    protected static String[] validCommands = {"mark", "m", "unmark", "um", "todo", "t", "deadline", "d", "event", "e", "list", "ls", "bye", "exit", "quit", "q"};
```

Comment:
Should use line wrapping instead, try to keep to a maximum of 120 characters per line.
Take note indentation for wrapped lines should be 8 spaces.